### PR TITLE
GitHub avatar

### PR DESCRIPTION
**Pull request by sweko**
Adds github as an avatar source, implementing #7

### CODE REVIEW

File: src/sources/utils.ts [hunk 1]
```diff
@@ -110,37 +111,54 @@ export function extractAsyncAvatarData(sourceType:string,data:any,size:number){
...
+
+/**
+* extract github avatar from json data
+*
+* @param {*} data
+* @returns url of the github avatar icon
+* @memberof AvatarComponent
+*/
+function _extractGithubAvatar(data: any, size?: number) {
+    if (!size) {
```
Comment (bhalperin) [inline src/sources/utils.ts]: Better: ```if (size) { ... } else { ... }```

Comment (HaithemMosbahi) [inline src/sources/utils.ts]: Yes I agree.

Comment (sweko) [inline src/sources/utils.ts]: Updated.

**Comment by HaithemMosbahi** [comment]
Great job @sweko . I think this PR is ready to be merged. Then I will release a new version that includes the support for Github as avatar source.
Thank you for your contribution

**Comment by sweko** [comment]
No problem, I have some more suggestions, that I'll open as enhancements.

**Pull request merged**